levels/inside_perimeter/events.js

```diff
@@ -263,6 +270,8 @@ module.exports = async function (event, world) {
     }
 
     openDoor("scroll_room_door");
+    // Forces the player's exclamation point to become hidden after opening the door
+    event.target.level.player.inRangeObject = null;
```

Comment:
Oh wow, this is wild. We don't have any sensible functionality to make an object _stop_ being in range of the player do we?

Reply:
Not yet, at least 😎